vnext/ReactUWP/Utils/PropertyUtils.h

```diff
@@ -273,67 +273,142 @@ bool TryUpdateCornerRadiusOnNode(
 }
 
 template <class T>
```

Comment:
why does this need to be a template? Isn't this always [a subclass of] winrt::Control ?

Reply:
Nope. TextBlock and TextElement inherit from FrameworkElement and DependencyObject, respectively.

---
In reply to: [371630526](https://github.com/microsoft/react-native-windows/pull/3830#discussion_r371630526) [](ancestors = 371630526)